rules/linux/credential_access_collection_sensitive_files.toml

```diff
@@ -24,37 +24,32 @@ note = """## Triage and analysis
 
 ### Investigating Sensitive Files Compression
 
-The rule identifies the use of compression utility to collect known files containing sensitive information,
-such as credentials and other system configurations.
+This rule identifies the use of compression utilities to archive known files that contain sensitive information,
+such as passwords, authorized ssh keys, vendor cloud configurations & credentials, encryption keys and system configurations.
 
 #### Possible investigation steps
 
 - Identify the session entry leader and session user.
 - Examine the contents of session leading to the compression activity via the Session View.
 - Examine the command execution pattern in the session, which may lead to suspicious activities.
 - Examine the command execution post sensitive file compressions to indicate any exfiltration of sensitive file data.
-- Investigate any other alerts or malicious activities associated with the user/host during the past 48 hours.
-
-### False positive analysis
-
-- System administrators performing regular back up of sensitive files.
-- Whitelisted data backup and recovery tools executing in sensitive file paths.
+- Examine the process creation events post sensitive file compressions to indicate any suspicious activities.
+- If the Analysis has shown exfiltration of sensitive file data, analyze the impact of data that has been exfiltrated.
```

Comment:
```suggestion
- If the analysis has shown exfiltration of sensitive file data, analyze the impact of data that has been exfiltrated.
```

Reply:
I think this also makes more sense in the "Response and remediation" section.

Reply:
https://github.com/elastic/detection-rules/pull/2872#discussion_r1239894932

It was in the Response Remediation section and was moved here based on 1st review.